packages/shared/src/fallbackClerk.ts

```diff
@@ -0,0 +1,50 @@
+import type { ClerkStatus } from '@clerk/types';
+
+/**
+ * Creates a minimal fallback Clerk instance when the main clerk-js script fails to load.
+ * This fallback is designed to be as simple as possible - it only provides the essential
+ * properties needed to trigger error UI components and prevent crashes.
+ */
+export function createFallbackClerk() {
+  const fallbackClerk = {
+    status: 'error' as ClerkStatus,
+    loaded: false,
+  };
+
+  return new Proxy(fallbackClerk, {
+    get(target, prop) {
+      if (prop in target) {
+        return target[prop as keyof typeof target];
+      }
+
+      if (typeof prop === 'string') {
+        if (prop.startsWith('is') || prop.startsWith('has')) {
+          return false;
+        }
+
+        if (
+          typeof prop === 'string' &&
```

Comment:
aren't we already in a block that only executes if `typeof prop` is string?

Reply:
Indeed!